Description: The user guide is installed in /usr/share/backup-manager-doc.
 This is the recommended canonical Debian location.
Author: Sven Joachim <svenjoac@gmx.de>
Bug-Debian: http://bugs.debian.org/433011
Bug-Debian: http://bugs.debian.org/528161

---
 doc/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/doc/Makefile
+++ b/doc/Makefile
@@ -1,7 +1,7 @@
 
 BASE	= user-guide
 SOURCE	= $(BASE).sgml
-DEST    = $(DESTDIR)/usr/share/backup-manager/doc
+DEST    = $(DESTDIR)/usr/share/doc/backup-manager-doc
 
 all: pdf txt html
 install: all
